sonic-thermalctld/scripts/thermalctld

```diff
@@ -565,16 +565,8 @@ class TemperatureUpdater(logger.Logger):
             phy_entity_keys = self.phy_entity_table.getKeys()
             for pek in phy_entity_keys:
                 self.phy_entity_table._del(pek)
-
-    def deinit(self):
-        """
-        Deinitializer of TemperatureUpdater
-        :return:
-        """
-        for name in self.temperature_status_dict.keys():
-            self.table._del(name)
-            if self.is_chassis_system and self.chassis_table is not None:
-                self.chassis_table._del(name)
+        if self.is_chassis_system and self.chassis_table is not None:
```

Comment:
this issue is not seen on fixed platform i.e non-modular chassis?

Reply:
not sure, need to investgate